### PR TITLE
initial_recipe

### PR DESCRIPTION
**Pull request by ritumd**
## Checklist before submitting

- [X] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by npanpaliya — COMMENTED** [review]
We don't need to add Apache License here. Keep LICENSE.txt from conda-forge and add IBM's copyright line in the same `Copyright (c) 2022, IBM`

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,10 +18,10 @@ build:
 requirements:
   host:
     - pip
-    - python
+    - python {{ python }}
```
Comment (cdeepali) [inline recipe/meta.yaml]:
```suggestion
    - python
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,10 +18,10 @@ build:
...
   run:
-    - python
-    - requests
+    - python {{ python }}
```
Comment (cdeepali) [inline recipe/meta.yaml]:
```suggestion
    - python
```

Comment (cdeepali) [reply recipe/meta.yaml]: We do not need python pin as this is `noarch:python `package.

**Comment by cdeepali** [comment]
I think we need to use --no-deps in the script field as in:

`script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv`

**Comment by ritumd** [comment]
Remote feedstock added

**Pull request closed without merging**